jupyterlab_notifications/handlers.py

```diff
@@ -105,23 +105,37 @@ async def post(self, path: str = ""):
                       NotifTimeout, NotifType, Created)
         cur.execute(
             "INSERT INTO notifs ( origin , Title ,Body , LinkURL ,Ephemeral , NotifTimeout , NotifType ,Created ) VALUES (? , ? ,? , ? ,? , ? , ? ,?)", insertData)
-
+        rowId = cur.lastrowid
         con.commit()
         con.close()
-        wsSend("yelooo")
+        self.set_status(201)
+        self.add_header("location", str(rowId))
+        # self.write(json.dumps({"RowId":str(rowId)}))
+        self.finish(json.dumps({"RowId":str(rowId)}))
+        self.flush()
+        # tornado.ioloop.IOLoop.make_current()
+        # time.sleep(3)
+        tornado.ioloop.IOLoop.current().spawn_callback(wsSend, str(rowId))
+        # wsSend(str(rowId))
 
 
 class notifyIDHandler(APIHandler):
     @tornado.web.authenticated
     async def get(self, notificationId):
-        con = sqlite3.connect('/mnt/f/git/jupyterlab-notifications/notif.db')
+        print(notificationId, "THIS WHAT I GOT")
+        con = sqlite3.connect('notif.db')
         cur = con.cursor()
 
 # Create table
         cur.execute('SELECT * FROM notifs where notificationId = ?',
                     (notificationId, ))
         data = cur.fetchall()
-        self.finish(json.dumps({"data": data}))
+        response = {}
+        for row in data:
+            response = {"INotificationResponse": {"notificationId": row[0], "origin": row[1], "title": row[2], "body": row[3],
```

Comment:
No need for outer "INotificationResponse" key (now object looks like {"INotificationResponse" : {"notificationId" : ..., ...} } instead of just {"notificationId" : ..., ...}